@xen-orchestra/web-core/lib/locales/en.json

```
@@ -110,9 +111,13 @@
   "core.sort.descending": "Sort descending",
   "core.textarea.exceeds-max-characters": "Field value must be {max} characters or less.",
   "cores-with-sockets": "Cores (sockets)",
+  "CPU-cap": "CPU cap",
+  "CPU-mask": "CPU mask",
```

```suggestion
  "cpu-cap": "CPU cap",
  "cpu-mask": "CPU mask",
```

No need to use uppercase in trad keys (for all keys)